web/app_options.js

```diff
@@ -127,7 +127,7 @@ const defaultOptions = {
   },
   renderInteractiveForms: {
     /** @type {boolean} */
-    value: false,
+    value: true,
```

Comment:
This change should be reverted because we should first test for a while and make sure that all form elements can be printed properly before enabling this for all users.